src/main/kotlin/com/openlattice/data/DataGraphIds.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2020. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+package com.openlattice.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.client.serialization.SerializationConstants
+import java.util.UUID
+
+data class DataGraphIds(
```

Comment:
@UnsungHero97 When are we planning to merge this PR/should I include those changes here? https://github.com/openlattice/api/pull/400

Reply:
That makes sense. We can close https://github.com/openlattice/api/pull/400 and just include those changes here.

Reply:
Cool, thanks! I wasn't sure, if that change is ready to merge/deploy.